circle.yml

```diff
@@ -0,0 +1,7 @@
+machine:
+  python:
+    version: 3.4.3
+
+test:
+  override:
+    - ./runtests.sh
```

Comment:
The auto test detection feature of pytest is troublesome.
In order to avoid it, we could invoke directly test modules or use other test tool.

BTW, [runtests.py is deprecated](https://pytest.org/latest/goodpractises.html#deprecated-create-a-pytest-standalone-script), so it is time to stop using runtests.py.